src/perl/dmtest/DMTest/Grouped.pm

```diff
@@ -0,0 +1,165 @@
+##
+# Base class for tests which run dmtest python tests.  Runs all the
+# tests quickly grouped into a single testcase.
+#
+# $Id$
+##
+package DMTest::Grouped;
+
+use strict;
+use warnings FATAL => qw(all);
+use Data::Dumper;
+use English qw(-no_match_vars);
+use Log::Log4perl;
+
+use Permabit::Assertions qw(
+  assertDefined
+  assertFalse
+  assertNumArgs
+  assertMinMaxArgs
+  assertRegexpDoesNotMatch
+  assertRegexpMatches
+);
+use Permabit::CommandString::DMTest;
+use Permabit::Utils qw(yamlStringToHash);
+
+use base qw(DMTest);
+
+my $log = Log::Log4perl->get_logger(__PACKAGE__);
+
+#############################################################################
+# @paramList{getProperties}
+our %PROPERTIES
+  = (
+     # @ple the starting device setup on the machine
+     dmtestRepo      => "git+ssh://bjohnston\@vdo-builder-bjohnston/~/git/dmtest-python",
```

Comment:
This says to get at the repo we should log into your account, probably not what you want :-)

Reply:
oh shoot. will fix

Reply:
fixed. as well as another fix relating to the actual git clone line